### PR TITLE
Remove unnecessary semicolon.

### PR DESCRIPTION
**Pull request by arturobernalg**
## Impact

- [ ] Bug fix (non-breaking change which fixes expected existing functionality)
- [X ] Enhancement/New feature (adds functionality without impacting existing logic)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Description

Remove any unnecessary semicolons inside block statements and/or duplicated.

## Things to be aware of

- Describe the technical choices you made
- Describe impacts on the codebase

## Things to worry about

- List any questions or concerns you have with the change
- List unknowns you have

### CODE REVIEW

**Comment by nvoxland** [comment]
**Initial/Pre-Review Thoughts**

Cleaning up the code, shouldn't impact any functionality

Questions I have:
- None

Potential risks:
- None

What could make the full review difficult:
- Nothing

**Review by MalloD12 — APPROVED** [review]
**Approved**.

Thank you @arturobernalg again for another code clean up PR. You are making look this code very good. Changes look ok to me (build and test passed successfully).

File: liquibase-core/src/main/java/liquibase/io/EmptyLineAndCommentSkippingInputStream.java [hunk 1]
```diff
@@ -42,7 +42,7 @@ private int read(final int lastRead, final boolean lookAhead) throws IOException
         if (commentSkipEnabled && (read == this.commentLineStartsWith.toCharArray()[0])
                 && (lastRead == '\n' || lastRead < 0)) {
             while ((((read = super.read())) != '\n') && (read != '\r') && (read > 0)) {
-                ;//keep looking
+                //keep looking
```
Comment (XDelphiGrl) [inline liquibase-core/src/main/java/liquibase/io/EmptyLineAndCommentSkippingInputStream.java:45]: **Today I Learned**

The `;` in this loop indicated an empty statement; Java still implicitly "knows" there is an empty statement, even without the `;` character.

File: liquibase-core/src/main/java/liquibase/diff/output/changelog/core/UnexpectedTableChangeGenerator.java [hunk 2]
```diff
@@ -47,7 +47,7 @@ public Change[] fixUnexpected(DatabaseObject unexpectedObject, DiffOutputControl
 
         for (Column column : unexpectedTable.getColumns()) {
             control.setAlreadyHandledUnexpected(column);
-        };
+        }
```
Comment (XDelphiGrl) [inline liquibase-core/src/main/java/liquibase/diff/output/changelog/core/UnexpectedTableChangeGenerator.java:50]: **Today I Learned** ... Java for loops don't require a closing `;`.  @arturobernalg, I'm learning a lot of interesting Java nuances with your code cleanup PRs!

Comment (XDelphiGrl) [reply liquibase-core/src/main/java/liquibase/diff/output/changelog/core/UnexpectedTableChangeGenerator.java:50]: Lol ... also, please tell me if I learned the wrong thing! I'm not a Java dev by trade.

File: liquibase-cdi/src/main/java/liquibase/integration/cdi/SchemesCDIConfigBuilder.java [hunk 3]
```diff
@@ -190,7 +190,7 @@ CDILiquibaseConfig fileLocked(final String id, Callable<CDILiquibaseConfig> acti
         FileLock lock = null;
         try (
                 FileOutputStream fileStream = new FileOutputStream(lockPath);
-                FileChannel fileChannel = fileStream.getChannel();
+                FileChannel fileChannel = fileStream.getChannel()
```
Comment (XDelphiGrl) [inline liquibase-cdi/src/main/java/liquibase/integration/cdi/SchemesCDIConfigBuilder.java:193]: **TIL** ... try-with-resources does not require a `;` in a single resource statement.

**Review by XDelphiGrl — APPROVED** [review]
This PR cleans up extraneous `;` characters in `try-with-resouces` and looping logic. 
* Functional and test harness executions passing.
* No additional testing required.

APPROVED

**Review by filipelautert — APPROVED** [review]

**Pull request merged**